### ISSUE TITLE
Add a GitHub corner to the website

### ISSUE THREAD

**Issue by wooorm** [get-alex/alex#200]
### Subject of the issue

There’s one small link on the website, to GitHub, but a [GitHub Corner](http://tholman.com/github-corners/) would be clear.

### Steps to reproduce

Go to [the website](https://alexjs.com)

### Expected behaviour

There should be a clearly visible way to go to GitHub.

### Actual behaviour

There is no such reference 😢

**Comment by Redarcher9** [comment]
Hi there,I'm new to open-source community.Can I work on this issue?

**Comment by wooorm** [comment]
Hi @Redarcher9! Yes, please!
I’d suggest waiting till tomorrow to participate in [hacktoberfest](https://hacktoberfest.digitalocean.com) though! 🎃

**Comment by Redarcher9** [comment]
Thankyou @wooorm !I am familiar with javascript but haven't used Node.js before.Can you help me figure out what things do i need to learn before contributing to this issue?

**Comment by wooorm** [comment]
If you’ve installed Node already, you should be fine following the [contributing guides](https://github.com/get-alex/alex/blob/master/contributing.md).

Note that the website is on the `website` branch, and the file you‘re looking for to add the corner is `src/index.html`. If you change that, and run `npm test`, all files are minified and built to the `dest` directory. If that looks file, you’re done! 👍

**Comment by meetcric** [comment]
Hi there,Can i work on this issue?

**Comment by wooorm** [comment]
@meetcric Let’s give @Redarcher9 a shot first!

**Comment by meetcric** [comment]
@wooormI think @cunhazera is working in #199. should I start to work on that issue?

**Comment by wooorm** [comment]
I believe he misplaced his comment, so I think you're free to work on that one!

**Comment by cunhazera** [comment]
@meetcric no, I'm not. That's all yours :)

**Comment by meetcric** [comment]
@wooorm can u explain this issue in details please?

**Comment by svcastaneda** [comment]
@TJScalzo I don't think that idea complies with [GitHub's logo usage guide](https://github.com/logos) mostly because of the animation and _possibly_ the usage of a purple background instead of a white or dark grey color

**Comment by TJScalzo** [comment]
@svcastaneda Does that mean the current GitHub corner is breaking their branding standards? The person who designed the corner seems to show it being used in a variety of color combinations on [his site](http://tholman.com/github-corners/). If the animated corner goes against GitHub’s logo rules, should we be using it at all?
